test/winter_onboarding_2021/fleet_management/db/cab_test.clj

```diff
@@ -0,0 +1,60 @@
+(ns winter-onboarding-2021.fleet-management.db.cab-test
```

Comment:
If we are keeping `models` in the `src` then I think our `test` directory too should reflect the `src` structure. 

Currently, these tests location reflect that the functions are in `src/.../db/cab.clj`